README.md

```diff
@@ -1,83 +1,87 @@
+* via Bitcoin [1GEnrRVobFYuqYj2opdvRQNtG3Z8znvfKw](bitcoin:1GEnrRVobFYuqYj2opdvRQNtG3Z8znvfKw?label=Donation%20for%20Sieve%20Addon)
 
 
 ## Questions and Bugs
 
-The best for questions is the public mailing list at https://groups.google.com/forum/#!forum/sieve-app or via mail sieve-app@googlegroups.com . But you can also send a private mail to schmid-thomas at gmx.net.
+The best place to ask questions is on the public mailing list at https://groups.google.com/forum/#!forum/sieve-app or by emailing sieve-app@googlegroups.com. However, you can also send a private email message to `schmid-thomas at gmx.net`.
 
-Concerning bugs please use the [issue tracker](https://github.com/thsmi/sieve/issues)
-or send a private email to schmid-thomas at gmx.net . You find more details on reporting bugs
-in the [Contributing Guidelines](https://github.com/thsmi/sieve/blob/master/CONTRIBUTING.md)
+Concerning bugs please use the [issue tracker](https://github.com/thsmi/sieve/issues) or send a private email to `schmid-thomas at gmx.net`. You find more details on reporting bugs in the [Contributing Guidelines](https://github.com/thsmi/sieve/blob/master/CONTRIBUTING.md)
 
 
-Give me 1-2 weeks time for a reply. If you did not receive a reply at all, it
-might be a good idea to check your spam filter.
+Please allow 1-2 weeks for a reply. If you did not receive a reply at all, it might be a good idea to check your spam filter.
+
 
 ## License
 
-The extension is free and open source software, it is made available to you
-under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
+The [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) is free and open source software, it is made available to you under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
 
 Refer to [Licensing information](https://github.com/thsmi/sieve/blob/master/LICENSING_INFO.md) for details.
 
+
 ## Releases
 
-You are looking for the most recent release?
+You are looking for the most recent [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) release?
 
-Just go to [the releases page and download the latest stable xpi or view the release notes](https://github.com/thsmi/sieve/releases/latest).
+[Available by downloading the latest stable xpi at the Releases page (release notes can also be viewed)](https://github.com/thsmi/sieve/releases/latest).
 
-**Firefox user have to do a right click on the XPI download and select "Save link as" otherwise Firefox will steal the link and try to install the addon it into Firefox which will fail**
+**Firefox users: At the [ATN Sieve add-on page](https://addons.thunderbird.net/addon/sieve/), right-click on the “+ Download Now” button and select “Save Link As…”; otherwise Firefox try to install the add-on into Firefox which will fail.**

```

Comment:
This note was not about thunderbirds addon page it is about the Release Page on github. You need to do there the rightclick stuff. Addons.thunderbird.net shows their own warnings directly before downloading. 

None except thunderbird extension developers know what ATN should mean.

Reply:
Will change as you requested.